CMakeLists.txt

```diff
@@ -85,7 +85,9 @@ endif()
 # modules and definitions
 #------------------------------------------------------------------------------
 include(cmake/utils.cmake)
-find_package(Maya 2018 REQUIRED)
+if (BUILD_MAYAUSD_LIBRARY)
+    find_package(Maya 2018 REQUIRED)
+endif()
```

Comment:
I added bunch of cmake changes to make UsdUfe buildable on its own. This library only needs Usd & Ufe (no Maya, etc).